examples/TrainEmbedding.py

```diff
@@ -3,25 +3,29 @@
 
 import boto
 s3_conn = boto.connect_s3()
+
 import multiprocessing
 cores = multiprocessing.cpu_count()
 
 import pandas as pd
 from skills_utils.time import datetime_to_quarter
 
+from skills_ml.job_postings.common_schema import JobPostingGenerator
```

Comment:
All this talk about examples makes me think we should create a JobPostingGenerator subclass with no dependencies, usable as an example. Maybe based on the Virginia dataset, downloading some subset of the data freshly, or maybe even we bundle like 100 job postings as an example (though maybe keeping the library small is better than keeping it example usable without an internet connection).

This is totally outside of the scope here, it just comes to mind as I'm reading the examples.

Reply:
yeah totally, but I think it could be in another PR.